sunpy/timeseries/sources/eve.py

```diff
@@ -50,11 +152,11 @@ class EVESpWxTimeSeries(GenericTimeSeries):
 
     References
     ----------
-    * `SDO Mission Homepage <https://sdo.gsfc.nasa.gov/>`_
-    * `EVE Homepage <http://lasp.colorado.edu/home/eve/>`_
-    * `Level 0CS Definition <http://lasp.colorado.edu/home/eve/data/>`_
-    * `EVE Data Acess <http://lasp.colorado.edu/home/eve/data/data-access/>`_
-    * `Instrument Paper <https://doi.org/10.1007/s11207-009-9487-6>`_
+    * `SDO Mission Homepage <https://sdo.gsfc.nasa.gov/>`__
```

Comment:
I think they all can be just a single underscore.

Reply:
oh I changed it as @Cadair had mentioned it should be __ - which should it be?

Reply:
In that case, ignore me.

Reply:
double underscores are generally a better idea for URLs.